data/brands/office/security.json

```
@@ -4,6 +4,20 @@
     "exclude": {"generic": ["^security$"]}
   },
   "items": [
+    {
+      "displayName": "Securitas",
+      "id": "securitas-1f2ee5",
+      "locationSet": {"include": ["001"]},
+      "tags": {
+        "brand": "Securitas AB",
```

```suggestion
        "brand": "Securitas",
```
Most OSM mappers prefer to not include the "inc", "ag", "ab", "gmbh" sorts of things, because that's just kind of a detail of how the company is registered.